modules/code_sandbox_worker.js

```diff
@@ -0,0 +1,42 @@
+/**
+* Worker script that runs a JavaScript code inside a sandbox.
+*
+* This script doesn't attempt to capture any errors or
+* prevent mishaps such as infinite loop as they will be handled by the manager.
+*
+* Instead, the focus is to just run the sandboxed code as efficiently as possible. 
+**/
+global.acorn            = require('../lib/js_interpreter/acorn_interpreter.js');
```

Comment:
why do we need a global variable here?

Reply:
From what I could gather from experimenting, the global variable is just required for the library to work.

The [example code](https://neil.fraser.name/software/JS-Interpreter/demos/node.js) written by the library author uses a global variable too and I wasn't able to get it to work otherwise.

Although it's not ideal, I thought this global variable wouldn't be a problem as it's isolated in a script that will be run as child processes of the sandbox manager, which limits its scope.